### DISCUSSION TITLE
https://cucoders.dev/publicaciones/ppmaluch/escuchando-los-cambios-en-los-datos-de-tablas-en-postgresql/

### DISCUSSION THREAD

**Discussion by giscus** [CuCodersCommunity/cucoderscommunity.github.io#49] [Page Comments]
# https://cucoders.dev/publicaciones/ppmaluch/escuchando-los-cambios-en-los-datos-de-tablas-en-postgresql/

Cómo podemos escuchar y reaccionar a eventos en la base de datos que creen o modifiquen datos, para luego ser procesados o manipulados.

https://cucoders.dev/publicaciones/ppmaluch/escuchando-los-cambios-en-los-datos-de-tablas-en-postgresql/

**Comment by manuelernestog** [comment]
Aseeeere el primer post con código de la comunidad, tiza 👏👏👏👏👏👏

**Reply by ppmaluch** [reply]
Jaja gracias hermano, ojalá vengan muchos más

**Reply by manuelernestog** [reply]
Seguro bro. Acabo de terminar de leerlo. Buenísimo 💪 

Pd: Hacia rato que no leía PSQL, uno con esto de los ORM se malacostumbra cantidad.

**Reply by alecorodri** [reply]
por fin salió a producción 🙌🏻👏🏻

**Reply by manuelernestog** [reply]
Si bro al fin, dio pincha pero ya salió 😁🙌🙌

**Reply by irnel** [reply]
excelente artículo, cuando sale el próximo :)

**Reply by manuelernestog** [reply]
Hay que ver cuando @ppmaluch se embulla de nuevo :)
